recipes/install.rb

```diff
@@ -68,6 +68,35 @@
 
 directory '/etc/mesos-chef'
 
+case node['mesos']['init']
+when 'systemd'
+  node.default['mesos']['master']['env']['ULIMIT'].gsub(/[^0-9]/, '')
```

Comment:
From my work with chef I would say it is better in the recipe (because it is within the runtime as if it wouldn't be within the attributes files.